emc-metalnx-services/src/test/java/com/emc/metalnx/services/unittests/AllTests.java

```diff
@@ -0,0 +1,12 @@
+package com.emc.metalnx.services.unittests;
+
+import com.emc.metalnx.services.irods.CollectionServiceImplTest;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({ CollectionServiceImplTest.class })
+public class AllTests {
+
+}
```

Comment:
Added unit tests.  The tests for emc-metalnx-services was not running and I believe it was because there was not an AllTests.java in place.  I added it and the CollectionServiceIplTest tests are now running.

Please take a look at this.  Also note that this isn't the only class in the emc-metalnx-services.  For now I wanted to reduce the scope of the change in case there was a reason these were omitted but I can add the other classes if we think it is necessary.

Reply:
I'm not super familiar with how this is meant to work.

@michael-conway Do you have any thoughts about this?

Reply:
After comparing this to other source files in this repo and reviewing the annotations, I think this is correct.